daffodil-runtime1-unparser/src/main/scala/org/apache/daffodil/processors/unparsers/SuppressableSeparatorUnparser.scala

```diff
@@ -136,28 +137,6 @@ final class SuppressableSeparatorUnparserSuspendableOperation(
     }
   }
 
-  /**
-   * Given two DataOutputStream, determine the set of data output streams starting from the first,
-   * and in chain until (and including) we reach the second.
-   *
-   * If the two are the same DataOutputStream, we get back a sequence of just the one DOS.
-   */
-  private def getDOSFromAtoB(beforeDOS: DataOutputStream, afterDOS: DataOutputStream): Seq[DataOutputStream] = {
```

Comment:
Did you move this to another file and forget to add it? It looks like it's stilll used above but I don't see it added back anywhere.